tests/unittests/general/web_middleware_test.py

```diff
@@ -29,6 +29,9 @@ def set_expiry(self, *_):
     def save(self, *_):
         pass
 
+    def cycle_key(self, *_):
+        pass
+
```

Comment:
Actually, this might be a good opportunity to consolidate these two identical mock class implementations.

Reply:
I can make a new PR for this as well

Reply:
#2841